### PR TITLE
Update to ZenPackLib v2

### PR DESCRIPTION
**Pull request by addjuarez**

### CODE REVIEW

**Comment by addjuarez** [comment]
Nvm dont review yet

**Comment by cluther** [comment]
@addjuarez: Let me know when this is ready for review.

**Comment by addjuarez** [comment]
@cluther Ready for review

File: README.rst [hunk 1]
```diff
@@ -607,6 +607,7 @@ Changes
 2.3.0
 
 - The zenoss.cmd.linux.rpm modeler plugin is now disabled by default. (ZPS-1603)
+- Converted to use zenpacklib v2
```
Comment (cluther) [inline README.rst]: The _Changes_ documentation is for users of the ZenPack. We should only document changes that affect the user in some perceivable way, and we should document what that perception would be. Most of our users have no idea what the implication of converting to zenpacklib v2 is.

Something more like the following might be more appropriate.idea

> Added ZenPackLib requirement. (ZPS-3000)

You should also update the _Releases_ section to add 2.3.0 so you can specify the new requirement on the ZenPackLib ZenPack there. See the following example from Docker where additional requirements for the release are documented following the versions of Zenoss with which it's compatible.

https://github.com/zenoss/ZenPacks.zenoss.Docker/blob/bda0307e637c815fc5c2c4d2be47e94ec707914c/README.html#L32

**Comment by cluther** [comment]
Please rebase against develop and squash the commits before merging.

**Pull request merged**